libshaderc/include/shaderc.h

```diff
@@ -24,12 +24,26 @@ extern "C" {
 #include <stdint.h>
 
 typedef enum {
-  shaderc_glsl_vertex_shader,
-  shaderc_glsl_fragment_shader,
-  shaderc_glsl_compute_shader,
-  shaderc_glsl_geometry_shader,
-  shaderc_glsl_tess_control_shader,
-  shaderc_glsl_tess_evaluation_shader
+  // Forced shader kinds. These shader kinds force the compiler to compile the
+  // source code as the specified kind of shader.
+  shaderc_glsl_forced_vertex_shader,
```

Comment:
Oops, I see, you mean to rename the forced ones to shaderc_glsl_<xxxx>_shader... I thought you want me to remove the default ones. Sorry, my mistake. So we should rename the forced ones to shaderc_glsl_<xxx>_shader, and keep the naming of the default ones?